spec/features/publish_interface/edit_provider_details_spec.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "About Your Organisation section" do
+  scenario "Provider user edits provider details" do
+    given_i_am_a_provider_user
+    and_my_provider_has_accrediting_providers
+    when_i_visit_the_details_page
+    then_i_can_edit_info_about_training_with_us
+    then_i_can_edit_info_about_our_accredited_bodies
+    then_i_can_edit_info_about_disabilities_and_other_needs
+    # TODO: then_i_can_edit_info_about_visa_sponsorship
+    then_i_can_edit_contact_details
+  end
+
+  def given_i_am_a_provider_user
+    @current_user = create(:user, :with_provider)
+    @provider = @current_user.providers.first
+    # Mark visa fields as false - switch to true when implementing visa section
+    @provider.update!(can_sponsor_student_visa: false, can_sponsor_skilled_worker_visa: false)
+    user_exists_in_dfe_sign_in(user: @current_user)
+  end
+
+  def and_my_provider_has_accrediting_providers
+    @provider.courses << create(:course, :with_accrediting_provider)
+    @accrediting_provider = @provider.accrediting_providers.first
+  end
+
+  def when_i_visit_the_details_page
+    visit details_publish_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year)
+    click_button "Sign in using DfE Sign-in"
+  end
+
+  def then_i_can_edit_info_about_training_with_us
+    click_link "Change details about training with your organisation"
+    fill_in "Training with you", with: ""
```

Comment:
I think we've opted to use page objects to drive these interactions (there should already be some for reference) would be good if we could stick to this going forward? Otherwise it'll be more tech debt to deal with later